src/http/params.cr

```diff
@@ -291,6 +296,11 @@ module HTTP
       raw_params.delete(name)
     end
 
+    # Whether the collection contains any params.
+    def empty?
+      raw_params.empty?
```

Comment:
You can use `delegate empty?, to: @raw_params`

Reply:
You can look at https://crystal-lang.org/api/master/Object.html#delegate%28%2Amethods%2Ctoobject%29-macro

Reply:
I knew that and i use it much in my project. use this because other methods not use delegate. so...i made a mistake 🤣